core/src/main/java/com/wttech/aem/acm/core/code/CodeShell.java

```diff
@@ -0,0 +1,23 @@
+package com.wttech.aem.acm.core.code;
+
+import groovy.lang.GroovyShell;
+
+public class CodeShell {
+
+    private final GroovyShell groovyShell;
+
+    private final CodeBinding codeBinding;
+
+    public CodeShell(GroovyShell groovyShell, CodeBinding codeBinding) {
+        this.groovyShell = groovyShell;
+        this.codeBinding = codeBinding;
+    }
+
+    public GroovyShell getGroovyShell() {
+        return groovyShell;
+    }
+
+    public CodeBinding getCodeBinding() {
```

Comment:
the purpose of introducing this to read codeBinding.getArguments() after invoking a method from script